roles/loki/promtail/tasks/config.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+- name: Backup configuration
```

Comment:
Missing tag `backup` like https://github.com/TOSIT-IO/tdp-collection/blob/8c1a55ba6b8dbad10646148e16905c659eccb50b/roles/hdfs/namenode/tasks/config.yml#L14

Reply:
Done in latest commit